manifests/init.pp

```diff
@@ -160,6 +163,7 @@
 #   Add --backtrace to systemd-coredump call systemd-coredump@.service unit
 #
 class systemd (
+  Pattern['^.+\.target$']                             $default_target = 'multi-user.target',
```

Comment:
Per se this is a breaking change (people with a non-default config will lose it). The title of the PR makes me think the default should be `undef` and only manage the default target if a value is specifically set.

I think we should adjust the PR title or the code so that they match :wink:

Reply:
Sure, but then we need a `Boolean $manage_default_target = true` to allow users who do *not* want to manage this to do as they which by setting it to false explicitely.

I trend to prefer a single parameter that accept undef and is not a breaking change :shrug: